src/frontend-samples/swiping-viewport/SwipingComparisonApi.ts

```diff
@@ -0,0 +1,265 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { BeEvent } from "@bentley/bentleyjs-core";
+import { ClipPlane, ClipPrimitive, ClipVector, ConvexClipPlaneSet, Point3d, Transform, Vector3d } from "@bentley/geometry-core";
+import { ContextRealityModelProps, FeatureAppearance, Frustum, RenderMode, ViewFlagOverrides } from "@bentley/imodeljs-common";
+import { AccuDrawHintBuilder, FeatureSymbology, GraphicBranch, IModelApp, IModelConnection, queryRealityData, RenderClipVolume, SceneContext, ScreenViewport, TiledGraphicsProvider, TileTreeReference, Viewport } from "@bentley/imodeljs-frontend";
+
+export enum ComparisonType {
+  Wireframe,
+  RealityData,
+}
+
+export default class SwipingViewportApp {
```

Comment:
File names is SwipingComparisonApi and class name is SwipingViewportApp  Please make these agree.

Reply:
fixed